filter-effects-2/Overview.bs

```diff
@@ -50,28 +54,281 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
+There are essentially two other choices that might be used as a definition of
+the "backdrop": 1) “everything that painted before, up to the root node”, or 2)
+“everything up to the parent stacking context”. However, there are ambiguities
+with definition #1, and excessive limitations with definition #2. Therefore, the
+Backdrop Root concept strikes a balance between the two.
+
+In particular, it is important to note that including “everything that paints
+before” an element is ambiguous in the case where an ancestor of the element
+contains filtering and/or opacity. Because those effects are inherited by
+descendant elements, including the element containing backdrop-filter or
+mix-blend-mode, it is not clear “when” to apply the effect. Filters and opacity
+create a stacking context, and the <a
+href="https://drafts.fxtf.org/filter-effects/#FilterProperty">css filter-effect
+specification</a> states that “All the [Stacking Context] descendants are
+rendered together as a group with the filter effect applied to the group as a
+whole.” If, somewhere inside that stacking context, an element contains a
+backdrop-filter property other than “none”, then it is impossible to
+honor the preceding sentence. At the point of the backdrop-filtered element, all
+of the (partially-painted) contents of the stacking context need to be fully
+rendered, including applied filters, opacity, and blending with the backdrop,
+and the resulting image needs to be used as the input to the backdrop-filtered
+element. That fundamentally breaks the atomicity of the stacking context. And the ambiguity
+arises from the need, at the end of rendering the fully-completed stacking
+context, to apply those filters and opacity <b>again</b> to the completed rendering.
+The filters and opacity will, at this point, be applied twice to the portion of
+the image that has been backdrop-filtered. This situation grows exponentially
+worse if backdrop-filters are nested inside each other. For this reason, it is
+necessary to prevent the backdrop-filter from “seeing" above nodes that have
+filters, opacity, and the other conditions listed above in the definition of the
+Backdrop Root.
+
+Performance would also be an issue, if backdrop-filter and mix-blend-mode were
+defined to filter "everything" that comes before them on the page. Each
+application of backdrop-filter or mix-blend-mode would require a separate
+rendering pass, to temporarily finish any partially-complete stacking contexts,
+and get the "final" output that would appear behind each element. That would
+double the required rendering time, and would potentially require twice the
+memory usage and GPU bandwidth to store the intermediate graphics texture
+holding the contents to be filtering. And assuming nested backdrop-filters or
+mix-blend-mode elements were allowed, this doubling would become an exponential
+performance breakdown. One set of nested backdrop-filtered elements would
```

Comment:
Remove sentence beginning with "one set...". The exponential sentence suffices.

Reply:
Done. I removed two sentences here.